js/Storages/ProjectStorage.js

```diff
@@ -23,10 +23,17 @@ SPM.Storages.ProjectStorage = {
 		return projects;
 	},
 
-	getMyProjects: function (projects) {
-		return (this.myProjects)?
-			Promise.resolve(this.myProjects.map(function (projectId) { return this.projects[projectId] }.bind(this))):
-			Promise.reject("No data")
+	getProjectsByUser: function (user) {
+        var myProjects = _.filter(this.projects, function(project) {
```

Comment:
I'm no fan of using thirs-party library function when their native counterpart exists already. Why not use `project.filter` `project.members.map` ? That's not really important, but I think it keeps the code clearer :)